condor_history_to_elasticsearch.py

```diff
@@ -10,6 +12,9 @@
 import classad
 
 import requests
+from elasticsearch import Elasticsearch
```

Comment:
Can we contain this import such that it's not imported if not needed?  Collecting the info from the collections doesn't need this, and it might not be available in the environment.

Reply:
Yep, good idea.